src/pybind/rados/rados.pyx

```diff
@@ -2692,6 +2693,38 @@ cdef class Ioctx(object):
             raise make_ex(ret, "failed to compare %s" % object_name)
         return completion
 
+    @requires(('object_name', str_type), ('xattr_name', str_type), ('oncomplete', opt(Callable)))
```

Comment:
we've dropped the `requires` decorator` in favor of the PEP484 style annotations. see bbf9f5bbe32b37bb1b79bbca4b9cc726dbb2cf34

Reply:
Thank you for your suggestion, update all in the evening.